### ISSUE TITLE
[question] can `scm` be inherited from a python_requires when using `scm_to_conandata`?

### ISSUE THREAD

**Issue by DukeOfKirkcaldy** [conan-io/conan#7236]
Not sure if this is a bug or not, I didn't see this listed as a [limitation](https://docs.conan.io/en/latest/extending/python_requires.html#limitations).

I want many of my packages to capture scm info from the working copy in the same way, so I added this code into a `python_requires` base.

```
scm = {
    "type": "git",
    "url": "auto",
    "revision": "auto",
    "username": tools.get_env("GIT_USER"),
    "password": tools.get_env("GIT_PASS")
}
```

This seems to be fine for storing scm info - I get all the right information in the right conandata.yml files.  When rebuilding these packages from sources, however, it seems something happens with the loading that conan will always use the scm info of the last loaded conanfile.py for all of the packages which have inherited 'scm'.

Is this something I might see a fix for, or shall it be added to known limitations?

**Comment by memsharded** [comment]
Could be a bug, or at least something to define as forbidden.

I am trying this possible approach to inherit the scm better in: https://github.com/conan-io/conan/pull/7238

**Comment by memsharded** [comment]
This is fixed, will be released in next 1.27. Thanks for reporting!
